_data/meltano/extractors/tap-theme-parks/danielpdwalker.yml

```
@@ -0,0 +1,25 @@
+capabilities:
+- state
+- catalog
+- discover
+- about
+- stream-maps
+description: Tap for theme park data.
+domain_url: https://themeparks.wiki/
+keywords:
+- api
+- meltano_sdk
+label: Theme Parks
+logo_url: /assets/logos/extractors/theme-parks.png
+maintenance_status: active
+name: tap-theme-parks
+namespace: tap_theme_parks
+pip_url: git+https://github.com/DanielPDWalker/tap-theme-parks.git
+repo: https://github.com/DanielPDWalker/tap-theme-parks
```

```suggestion
repo: https://github.com/DanielPDWalker/tap-theme-parks
quality: bronze
```
We added a new quality attribute. It will start to get refreshed on its own but bronze is a good starting place.